processor/cumulativetodeltaprocessor/processor.go

```diff
@@ -44,6 +42,20 @@ func newCumulativeToDeltaProcessor(config *Config, logger *zap.Logger) *cumulati
 	return p
 }
 
+func getMetricTypeFilter(types []string) map[pmetric.MetricType]bool {
```

Comment:
Can you check in `pmetric`, I think there already exists a way to go from the string name to the actualy `MetricType`.  It might be an unmarshaller.

Reply:
I checked, but could not find anything that provides this function - The unmarshaler there checks the existence of fields which are named according to the different metric types (https://github.com/open-telemetry/opentelemetry-collector/blob/main/pdata/pmetric/json.go#L114-L123) but it seems like there is no functions that directly returns the `MetricType` for a given string